src/tests/ftest/rebuild/rebuild_container_rf.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from container_rf_test_base import RbldContRedundancyFactor
+
+
+class RbldContRedundancyFactorrr(RbldContRedundancyFactor):
+    # pylint: disable=too-many-ancestors
+    """Test container redundancy factor with various rebuilds.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a Rebuild Container RF with ObjClass Write object."""
+        super().__init__(*args, **kwargs)
+        self.daos_cmd = None
+
+    def test_rebuild_with_container_redundancy_factor(self):
```

Comment:
very long name too ...

Reply:
Will shorten the name, please see next commit.